internal/server/proto/server.proto

```diff
@@ -332,6 +390,9 @@ message Project {
   // change.
   string file_change_signal = 8;
 
+  // Variable values stored on the server. These values are set in the UI.
+  repeated Variable variables = 9;
```

Comment:
I would modify the comment to add that these are still a work in progress, experimental, and so far do nothing so that if we make a releaes without variables complete, we aren't misleading folks who might be reading this.